src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -10302,7 +10302,11 @@ namespace AirflowNetwork {
                         ErrorsFound = true;
                     } else {
                         NodeFound(i) = true;
-                        AirflowNetworkNodeData(zoneNum).EPlusNodeNum = i;
+                        for (int iZone = 1; iZone <= AirflowNetworkNumOfZones; iZone++) {
+                            if (MultizoneZoneData(iZone).ZoneNum == zoneNum) {
+                                AirflowNetworkNodeData(iZone).EPlusNodeNum = i;
+                            }
```

Comment:
The zone number should be AFN zone index, instead of zone number from Zone object.

Reply:
OK I see it.  This accesses the right element of the AFN Node Data array.  Good.